### PR TITLE
Attempt at providing a noblas.

### PR DESCRIPTION
**Pull request by pelson**
Unfortunately numpy is still using accelerate locally, and I suspect it is also the case on the CI.

Useful script: https://gist.github.com/alimuldal/eb0f4eea8af331b2a890

### CODE REVIEW

**Comment by jakirkham** [comment]
It is. Let me try a PR that should help.

**Comment by jakirkham** [comment]
So, I'm thinking about this and honestly I would be ok with a `noblas` package. It would just be a no-op, but I think it would avoid dependent recipes from getting so gross.

**Comment by jakirkham** [comment]
Yeah, I think we will need a patch. Opened a bug report ( https://github.com/numpy/numpy/issues/7652 ) upstream. Let's see if they have any advice.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```
Comment (jakirkham) [inline recipe/meta.yaml]: Let's go ahead and bump this to `101`. Going forward I would like us to reserve 100-199 for `noblas` and 200-299 for `openblas`. Basically build number is playing a big role in terms of what gets installed. So, I'd like to keep it higher. Again pinning should give you what you want here.

Comment (msarahan) [inline recipe/meta.yaml]: ping @mcg1969 - this makes me cringe as a method of determining which variant is installed.  Please try to find a better way.

Comment (jakirkham) [inline recipe/meta.yaml]: It makes us cringe too, but unfortunately we had to get something ASAP so this is where @pelson and I are with it. Features with exclusions is what we really want to see.

Comment (mcg1969) [inline recipe/meta.yaml]: How ASAP is ASAP? We're doing channel priority handling for conda 4.1.

Comment (jakirkham) [inline recipe/meta.yaml]: Would have to ask @pelson, but I go the impression that it was highly desirable to have this out by today.

Comment (jakirkham) [inline recipe/meta.yaml]: I mean we can limp along with this for a little while. We tried to pick some that worked now, but should be pretty stable for awhile (even if it is hideous). That way we have some time to work on something better for `conda` and/or `conda-build`.

Comment (jakirkham) [inline recipe/meta.yaml]: Just to clarify what was needed ASAP was NumPy and by extension SciPy. We can continue to discuss `conda` and `conda-build` changes. What we have done here is not pretty, but it let's these packages come into existence in conda-forge without putting undue pressure on anyone else. We fully expect to change this in the long term.

**Comment by ocefpaf** [comment]
@pelson closing this b/c the rebase alone would be a nightmare. We should try to do this again at some point though.

**Pull request closed without merging**